### ISSUE TITLE
also download cover

### ISSUE THREAD

**Issue by Shikitsuna** [v613/toonily-dl#2]
if possible, download convers and put it into the chapter folders
so it will be like
\books\chapter x\images.png
\book\cover.png

**Comment by v613** [comment]
Thanks @Shikitsuna for a good idea.
If I understood you correctly, you mean
```mermaid
graph TD;
  Book-->Chapter1;
  Book-->ChapterX;
  Book-->Cover.png;
  
  Chapter1-->Image1;
  Chapter1-->ImageX;

  ChapterX-->image1;
  ChapterX-->imageX;
```

**Comment by Shikitsuna** [comment]
Correct , so if i imported to a file/app the cover woudln't be too long since usually apps take any possible file inside the folders, like tachi/hentoid/perfect viewer

**Comment by v613** [comment]
Implemented in release `v0.1.1`
